rholang/src/main/scala/coop/rchain/rholang/interpreter/Registry.scala

```diff
@@ -842,44 +844,79 @@ class RegistryImpl[F[_]](
       case _ => F.unit
     }
 
+  // TODO: Extract hardcoded id:<whatever> and maybe even pass in this resolver itself
+  object BlessedContract {
+    val idMap = HashMap[String, String](
```

Comment:
What is wrong with `Map.apply`?

Reply:
Does this field need to be publicly available?

Reply:
Can you please expand on this?

Reply:
I'm failing to see the benefit of using `HashMap` explicitly, instead of defaul `Map`